src/main/java/org/reactome/release/qa/check/ChimericInstancesCheck.java

```diff
@@ -69,6 +74,9 @@ protected void checkChimericInstances(QAReport report, String clsName) throws Ex
             if (isEscaped(rle)) {
                 continue;
             }
+            if (skipList.inSkipList(rle)) {
```

Comment:
Since “skipList” is only used here to call methods, the variable could be local here instead if a class variable.

Reply:
I think I am against that, for consistency's sake. For other classes, it needs to be a class variable because the skiplist check takes place in a method. If we make some local variables, and other global variables, it just makes the implementation between them a little different. 

That being said, if the preference is to have it as a local variable anyways for classes where it could be, I can implement that

Reply:
Okay, it’s fine with me to leave the skip list as class instance variables